packages/components/src/ClipboardButton/ClipboardButton.js

```diff
@@ -0,0 +1,45 @@
+import { contextConnect, useContextSystem } from '@wp-g2/context';
+import { useClipboard } from '@wp-g2/utils';
+import { noop } from 'lodash';
+import React from 'react';
+import { useEffect } from 'react';
+import { useRef } from 'react';
+
+import { Button } from '../Button';
+
+function ClipboardButton(props, forwardedRef) {
+	const {
+		text,
+		onClick: onClickProp = noop,
```

Comment:
I don't think we need to alias `onClick` to `onClickProp` here :)
(unlike `onCopy`, which comes from `useClipboard`)